### ISSUE TITLE
Barbiturates produce worsening

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4797]
**Preferred term label:**
exacerbated by barbiturate 

**Synonyms**
Barbiturates produce worsening

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal
adding as HP:0032502
